docs/how_to_contribute/math/how_to_add_prime_fields.md

```diff
@@ -0,0 +1,79 @@
+# How to add prime fields
+
+Follow this guide to add a new prime field for Tachyon.
+
+_Note_: We have our own development conventions. Please read the [conventions docs](/docs/how_to_contribute/conventions.md) before contributing.
```

Comment:
```suggestion
_Note_: We have our own development conventions. Please read the [conventions doc](/docs/how_to_contribute/conventions.md) 
```

Reply:
I'm so sorry ;-; I didn't mean to get rid of "before contributing."
it should be 
```suggestion
_Note_: We have our own development conventions. Please read [conventions docs](/docs/how_to_contribute/conventions.md) before contributing.